exporter/kinesisexporter/README.md

```diff
@@ -1,3 +1,64 @@
 # Kinesis Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics to Kinesis.
+
+The following configuration options are supported:
+
+- `queue_size`: Dictates how many spans the queue should support. default: `100000` 
```

Comment:
Which queue / workers are these? I guess the collector has queuing logic so it's hard to follow what these parameters actually control.

Reply:
I'm also a little confused here. It seems should used by exporter. but I can't see any test about it. I would really appreciate it If you can help to take a deep see on it. : )